pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java

```diff
@@ -414,7 +414,7 @@ private int consumeFromWebSocket(String topic) {
                     LOG.debug("No message to consume after waiting for 5 seconds.");
                 } else {
                     try {
-                        System.out.println(Base64.getDecoder().decode(msg));
+                        System.out.println(Arrays.toString(Base64.getDecoder().decode(msg))); // print decode
```

Comment:
Arrays.toString of a byte[] is still not very useful most of the times

in other commands in the pulsar-client we have an option to print byte[] in hex 
https://github.com/apache/pulsar/blob/27156e4866298c44e83724eeb10a4ee67317433c/pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java#L100

can you please implement the support for 'displayHex' ?

Reply:
Ok, I will do that : )

Reply:
I have added support for `displayHex`